docs/setup.md

```diff
@@ -0,0 +1,65 @@
+# Setup
+
+This launcher is intended for deployment as a sidecar container alongside an n8n main instance container, ideally both on the same host. The launcher exposes a health check endpoint at `/healthz` on port `5680` for liveness checks, and the n8n main instance does so on port `5681`. The orchestrator (e.g. k8s) can use these to monitor the health of the launcher and of the n8n main instance.
```

Comment:
Sidecar always runs on the same host. Also we recommend running one for each n8n container (main, worker).

```suggestion
This launcher is intended for deployment as a sidecar container alongside an n8n instance container(s). The launcher exposes a health check endpoint at `/healthz` on port `5680` for liveness checks. The orchestrator (e.g. k8s) can use this to monitor the health of the launcher and of the n8n main instance.

The runner should expose a health check endpoint at `/healthz` on port `5681` for liveness checks. The launcher will use this to endpoint to monitor the health of the runner, and terminate it if needed.
```